src/ui/tutorial/tutorial.gss

```diff
@@ -12,14 +12,17 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
+ *
+ * @author mdiehl@workbenchplatform.com (Matt Diehl)
```

Comment:
There is also an AUTHORS file under https://github.com/google/coding-with-chrome/blob/master/AUTHORS which could be modified.

Reply:
I added Matt because most of the new code in this file has just been moved from addons/workbench/workbench_project.js What do you prefer? The @author line in each file, the AUTHORS file or both?

Reply:
You could mention workbench as company  in AUTHORS and leave Matt here.
Only want to avoid a long list of authors, so in this case it makes sense. ;)